### ISSUE TITLE
Potential missing calls when using ngmlr for alignment

### ISSUE THREAD

**Issue by jamesdgalbraith** [fritzsedlazeck/Sniffles#533]
Hi,

This might not be a big problem but I just wanted to check a patch to an error in NGMLR might cause issues in Sniffles.

I'm using NGMLR for aligning PacBio Revio reads to a reference assembly before calling variants with Sniffles and I've run into an issue that I'm worried might result in missing variants. Using older versions of ngmlr such as the one currently available from bioconda results in numerous alignments with the 2064 flag (supplementary alignments + reverse strand, see https://github.com/philres/ngmlr/pull/96) being assigned a mapq score of -2147483648. This negative score leads to recent versions of samtools failing to parse the nglmr SAM into a BAM. The latest version of NGMLR avoids this issue by setting these alignments to have a MAPQ to 0, however I'm curious if this (or the negative values) could cause variants which would be visible in viewing the alignments in say IGV would to not be called due to scores below 20 are filtered out by default.

I just wanted to check, as in prior discussions on this Git repo I've see discussion that using minimap2 as the alignment will result in Sniffles calling false positives.

Many thanks,
James.

**Comment by fritzsedlazeck** [comment]
Yeah Sniffles right now filters for 20 or higher MQ... thats a parameter that you could change. We are not maintaining NGMLR anymore... sorry
Fritz
